java-tools/OsmAndServer/src/main/java/net/osmand/server/api/services/TrackAnalyzerService.java

```diff
@@ -388,4 +397,11 @@ public QuadRect calculateQuadRect(List<WptPt> points) {
 		}
 		return bbox;
 	}
+
+	public String[] getQuadTileShortlinks(QuadRect bbox) {
```

Comment:
This is not correct and not what we discsussed. It's still same problem that long route will appear in all possible unrelated searches cause it covers big bbox